packages/data/src/namespace-store.js

```diff
@@ -24,34 +24,16 @@ import createResolversCacheMiddleware from './resolvers-cache-middleware';
  * @return {Object} Store Object.
  */
 export default function createNamespace( key, options, registry ) {
-	// TODO: After register[Reducer|Actions|Selectors|Resolvers] are deprecated and removed,
-	//       this function can be greatly simplified because it should no longer be called to modify
-	//       a namespace, but only to create one, and only once for each namespace.
+	const reducer = options.reducer;
+	const store = createReduxStore( reducer, key, registry );
 
-	// TODO: After removing `registry.namespaces`and making stores immutable after create,
-	//       reducer, store, actinos, selectors, and resolvers can all be removed from here.
-	let {
-		reducer,
-		store,
-		actions,
-		selectors,
-		resolvers,
-	} = registry.namespaces[ key ] || {};
-
-	if ( options.reducer ) {
-		reducer = options.reducer;
-		store = createReduxStore( reducer, key, registry );
-	}
+	let selectors = {};
```

Comment:
These never defaulted to objects previously, do they need to?

Reply:
Yes, probably not impactful, I don't want to wait for the tests once more :P

Reply:
It could be good to enforce one way or the other by tests. I think it has an impact on what's returned from `select( 'store-not-exists' )`. What would you expect the return value to be?

In master, it's `undefined`. In this branch, it's now `{}`.

Impactful 😬

Reply:
really? I don't expect this change to impact that though :)

Reply:
It is still returning undefined for me.

Reply:
Oh, I guess it's:

```js
wp.data.registerStore( 'store-no-selectors', { reducer: () => null } );
wp.data.select( 'store-no-selectors' );
```

Master: `undefined`
This branch: `{}`

Reply:
I was going to add unit tests, but I actually think it may have been "improved" by the `{}` change being the expected behavior. Let's deal with it separately.

Reply:
(cc @coderkevin )

Reply:
I don't have a strong opinion either way on this. `{}` makes sense to me.